assets/prototype/domain/shared/predicates/shared/isTrashed/index.test.ts

```diff
@@ -1,26 +1,26 @@
 import { omit } from 'ramda';
 
-import isTrashed from './index';
-import { nodes as datetimes } from '../../../../../eventEditor/data/queries/datetimes/test/data';
+import isTrashed from './';
+import { nodes as datetimes } from '../../../../eventEditor/data/queries/datetimes/test/data';
 
 describe('isTrashed', () => {
-	it('should return false if isTrashed prop is not boolean', () => {
+	it('should return false if isTrashed prop is undefined', () => {
 		datetimes.forEach((datetime) => {
 			const newDatetime: any = omit(['isTrashed'], datetime);
 			const result = isTrashed(newDatetime);
-			expect(result).toBe(false);
+			expect(result).toBe(undefined);
```

Comment:
wonder if we should change the predicate so that this test description is accurate and `false` is returned instead of values like `undefined` or `null` ?

ie what is we changed:

```ts
const isTrashed = <T extends Trashable>(entity: T): boolean => {
	return entity.isTrashed;
};
```

to:

```ts
const isTrashed = <T extends Trashable>(entity: T): boolean => {
	return entity.isTrashed && entity.isTrashed === true;
};
```

or some other alternative to ensure that the only value that returns `true` is an actual boolean `true` and all others will return `false`

Reply:
Since we have made it (in fact all fields) required in TS, checking for its existence is unnecessary.

Reply:
but according to this test:

```ts
const result = isTrashed(newDatetime);
expect(result).toBe(undefined);
```

that field can be `undefined`